contrib/initramfs/scripts/zfs

```diff
@@ -327,7 +331,23 @@ mount_fs()
 
 	# Need the _original_ datasets mountpoint!
 	mountpoint=$(get_fs_value "$fs" mountpoint)
-	ZFS_CMD="mount -o zfsutil -t zfs"
+	atime=$(get_fs_value "$fs" atime)
```

Comment:
It sounds like the time stuff is to be removed and addressed separately?

Reply:
Indeed, looks like `atime`/`relatime` props issue can be solved at `mount.zfs` level https://github.com/openzfs/zfs/issues/7947